src/ripple/app/main/Application.cpp

```diff
@@ -280,7 +280,8 @@ class ApplicationImp : public Application, public RootStoppable, public BasicApp
         , pgPool_(
               config_->reporting() ? make_PgPool(
                                          config_->section("ledger_tx_tables"),
-                                         logs_->journal("PgPool"))
+                                         logs_->journal("PgPool"),
+                                         *this)
```

Comment:
I think the parent should be `ReportingETL` actually. `ReportingETL` depends on `PgPool`, and needs to be stopped after `PgPool` is stopped. `Application` is already a Stoppable parent of `ReportingETL`. We should also update the diagram in `Stoppable.h`

Reply:
I'm OK with this, also need to make sure that RPC also works with this arrangement.